openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -4763,7 +4763,14 @@ OS:Schedule:FixedInterval,
   A3, \field Schedule Type Limits Name
        \type object-list
        \object-list ScheduleTypeLimitsNames
-  A4, \field Interpolate to Timestep
+  A4, \field Translate to Schedule File
```

Comment:
@macumber @joseph-robertson Thanks for making this optional and defaulting to No.